activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeCandidatesTest.java

```diff
@@ -93,13 +101,73 @@ public void createStandaloneTaskAndDeleteAndAddUserCandidates() {
         .extracting(event -> event.getEntity().getUserId())
         .contains("garth",
                   "garth");
+
+        task = taskRuntime.task(createTask.getId());
+        userCandidatesOnTask = task.getCandidateUsers();
+        assertThat(userCandidatesOnTask).isNotNull();
+        assertThat(userCandidatesOnTask.size()).isEqualTo(1);
         
         userCandidates = taskRuntime.userCandidates(createTask.getId());
         assertThat(userCandidates).isNotNull();
         assertThat(userCandidates.size()).isEqualTo(1);
-           
      }
 
+    @Test
+    public void should_addAndRemoveCandidateGroup() {
+
+        RuntimeTestConfiguration.taskCandidateUserRemovedEvents.clear();
```

Comment:
Don't we have an @After test cleanup?
All good otherwise 👍

Reply:
The cleanup already exists at the beginning of the test::
```
 @After
    public void taskCleanUp(){
        taskCleanUpUtil.cleanUpWithAdmin();
    }
```